tests/test_request_response.py

```diff
@@ -32,7 +32,7 @@ def testRawCall(self):
         """
         keys = [self.domain]
         sess = RequestResponseHandling(store_keys=keys)
-        response = self.requests.post(*keys)
+        response = self.requests.get(*keys)
```

Comment:
What's the reasoning behind this change?

Reply:
The test was failing with a 403 error. 
I think that example.com domain does not let this kind of requests, handled in this way, anymore. 
Since it seems just a generic request test, and just changing the post in the get worked, I thought we can change it in this way.

Reply:
What about other methods? They all use `post`. Also the original `requests.post()` is backed up before test execution and restored afterwards, nothing like that is done for `requests.get()`.

> I think that example.com domain does not let this kind of requests, handled in this way, anymore.

I don't think it was a good idea to rely on that in the first place :sweat_smile: We should probably use a domain that we control.

Reply:
Should I change the test and use `packit.dev` with the `get`?

Reply:
I don't know much about requre, but these tests and the code they cover are clearly meant to handle POST requests.

Reply:
_mf is actually surprised that example.com is assigned, I thought it's literally meant for docs and local deployment for generating examples_

Reply:
`packit.dev` also doesn't accept POST requests, so we would need to make that work first. But I think that's the way to go.

Reply:
I have used https://stg.packit.dev/api/webhooks/github, to me it seems to work.